book/src/raw-block.md

```diff
@@ -11,22 +11,62 @@ A *mounted (file) volume* is volume that will be mounted using a specified file
+than mount volumes, because there's no filesystem state stored on the node side that creates any technical
+impediments to multi-attaching block volumes. There may still be good reasons to prevent multi-attaching
+block volumes, and there may be implementations that are not capable of supporting multi-attach, but it's
+generally wrong to use exactly the same set of access modes for block as for mount volumes.
+
+CSI plugins that support both mount and block volumes must be sure to check the capabilities of all CSI RPC
+requests and ensure that the capability of the request matches the capability of the volume, to avoid trying
+to do file-system-related things to block volumes and block-related things to file system volumes.
+
+Also, CSI plugins that implement the optional `GetCapacity()` RPC should note that that RPC includes
+capabilities too, and if the capacity for mount volules is not the same as the capacity for block
+volumes, that needs to be handled in the implementation of that RPC.
+
+Q: Can CSI plugins support only block volumes and not mount volumes?
+A: Yes! This is just the reverse case of supporting mount volumes only. Plugins may return `InvalidArgument`
+for any creation request with an `AccessType` of `VolumeCapability_Mount`.
+
+## Differences Between Block and Mount Volumes
+
+The main difference between block volumes and mount volumes is the expected result of the `NodePublish()`.
+For mount volumes, the CO expects the result to be a mounted directory, at `TargetPath`. For block volumes,
+the CO expect there to be a device file at `TargetPath`. The device file can by a bind-mounted device from
+the hosts `/dev` filesystem, or it can be a device node created at that location using `mknod()`.
+
+It's desirable but not required to expose an unfiltered device node. For example, CSI plugins based on
+technologoies that implement SCSI protocols should expect that pods consuming the block volumes they create
+may want to send SCSI commands to the device. This is something that should "just work" by default (subject
+to container capabilities) so CSI plugins should avoid anything that would break this kind of use case. The
+only hard requirement is that the device implements block reading/writing however.
+
+For plugins with the `RPC_STAGE_UNSTAGE_VOLUME` capability, the CO doesn't care exactly what is placed at
+the `StagingTargetPath`, but it's worth noting that some CSI RPCs are allowed to pass the plugin either
+a staging path or a publish path, so it's important to think carefully about how `NodeStageVolume()` is
+implemented, knowing that either path could get used by the CO to refer to the volume later on. This is
+made more challenging because the CSI spec says that `StagingTargetPath` is always a directory even for
+block volumes.
 
-Because this feature is still alpha it is disabled by default in Kubernetes. You must enable the feature on Kubernetes:
```

Comment:
We should still document the feature gates that control the behavior on Kubernetes even though it's enabled by default